_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Introduction of Recommender Algorithms for E-Commerce"
+date: "2015-09-16"
+icon: Light-bulb
+categories: tech-stories
+authors: ["Pujit"]
+---
+
+##Introduction
+
+Recommender systems in e-commerce can be seen as a sales personal.  Like a good sales personal in a store helps increasing sales and building loyalty customer base, recommender system handles sales three ways: browser into buyers, cross-sell and loyalty.
+
+* **Browsers into buyers:** Visitors to a Web site often look over the site without ever purchasing anything. Recommender systems can help customers find products they wish to purchase.
+
+* **Cross-sell:** Recommender systems improve cross-sell by suggesting additional products for the customer to purchase. If the recommendations are good, the average order size should increase. For instance, a site might recommend additional products in the checkout process, based on those products already in the shopping cart.
+
+* **Loyalty:** In a world where a site’s competitors are only a click or two away, gaining customer loyalty is an essential business strategy. [4,5] Recommender systems improve loyalty by creating a value-added relationship between the site and the customer. Sites invest in learning about their users, use recommender systems to operationalize that learning, and present custom interfaces that match customer needs. Customers repay these sites by returning to the ones that best match their needs. [3]
+
+Hence, it is not surprising that Recommender Systems are essential part of e-commerce solutions in this era.  At present day, many recommender algorithms exist which can be employed into recommender systems built for e-commerce.  In this blog-post we will have high level overview of the classification of such recommender algorithms.
+
+
+##Algorithms
+
+Recommender Algorithms are mainly classified into two categories as described below:
+
+1. ###Content based recommendation systems
+> Content-based recommendation systems recommend an item to a user based upon a description of the item and a profile of the user’s interests. While a user profile may be entered by the user, it is commonly learned from feedback and/or the interaction of the user on items. A variety of learning algorithms have been adapted to learning user profiles. [1]
+>
+> Algorithms: Relevance feedback, genetic algorithms, neural networks, and the Bayesian classifier are among the learning techniques for learning a user profile. The vector space representation and latent semantic indexing can both be used by these learning methods to represent documents.
+>
+> **Pros:** It does not rely on other user's feedback. Provides purely feature based recommendations. It Does not suffer from cold start.
```

Comment:
@steffimeier "cold start" is very typical problem of recommender systems.  When deploying such system first time (in e-commerce or any other domain), initially it does not have enough data to compute quality recommendations.  So it takes some time to gather enough information about buyers, their likings etc.  There are many research papers, articles, blogs, etc. available.